From c6ba8c7373e98eeb499a7a52ba2855e4148f3664 Mon Sep 17 00:00:00 2001
From: Bin Han <bin.b.han@intel.com>
Date: Fri, 10 Oct 2014 17:27:17 +0800
Subject: [PATCH] atomisp2: add IOCTL ATOMISP_IOC_G_INVALID_FRAME_NUM

BZ: 226088

This IOCTL is used for HAL to query how many invalid frames there will be at
the beginning of ISP pipeline output.

Change-Id: I38c167339558ae89133ba8425d90ac7c5b4b4156
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-2564
Signed-off-by: Bin Han <bin.b.han@intel.com>
Reviewed-on: https://android.intel.com:443/286414
---
 include/linux/atomisp.h |    2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/linux/atomisp.h b/include/linux/atomisp.h
index 3341784..e80b6a2 100644
--- a/include/linux/atomisp.h
+++ b/include/linux/atomisp.h
@@ -1190,6 +1190,8 @@ struct atomisp_sensor_ae_bracketing_lut {
 #define ATOMISP_IOC_S_SENSOR_AE_BRACKETING_LUT \
 	_IOW('v', BASE_VIDIOC_PRIVATE + 43, struct atomisp_sensor_ae_bracketing_lut)
 
+#define ATOMISP_IOC_G_INVALID_FRAME_NUM \
+	_IOR('v', BASE_VIDIOC_PRIVATE + 44, unsigned int)
 
 #define ATOMISP_IOC_G_EFFECTIVE_RESOLUTION \
 	_IOR('v', BASE_VIDIOC_PRIVATE + 45, struct atomisp_resolution)
-- 
1.7.9.5

